src/engine/controls/cuecontrol.cpp

```diff
@@ -1861,19 +2168,34 @@ HotcueControl::HotcueControl(QString group, int i)
 
 HotcueControl::~HotcueControl() {
```

Comment:
This file is enormous. Could you split HotcueControl into a separate file?

Reply:
Yes, but I'm unsure if I should do it now. Are any other active PRs touching cuecontrol.cpp?

Reply:
#2861 but I don't think it would be affected by this. I'm okay with deferring this to another PR if you add a checkbox for it to the top of this PR.

Reply:
It would be great if there was a tool that automatically checked for merge conflicts between open PRs.

Reply:
I won't, but'll add a TODO.